packages/open-payments/src/client/requests.ts

```diff
@@ -140,10 +178,13 @@ const handleError = (
   }
 
   const errorMessage = `Error making Open Payments ${requestType} request`
-  deps.logger.error({ status: errorStatus, errorDescription }, errorMessage)
+  deps.logger.error(
+    { status: errorStatus, errorDescription, url, requestType },
+    errorMessage
+  )
```

Comment:
This is probably quite the edge case, but `errorStatus` and `errorDescription` could be undefined if the error is not a validation error, and `HTTPError`, or `Error`. Just wondering if we can handle that better such as logging the original error or something.

Reply:
Yep, seems quite unlikely, but updated to log in any case :+1: